### PR TITLE
added documentation to update an active deployment

### PR DESCRIPTION
**Pull request by rsum1**

### CODE REVIEW

File: guides/deploy/README.md [hunk 1]
```diff
@@ -371,6 +371,107 @@ akash \
 
 where `$SERVICE_NAME` is the name of a service defined in your [SDL](/sdl).
 
+## Update your deployment
+
+Updating active deployments is a two step process. First, edit the deploy.yml with the desired changes.
```
Comment (hydrogen18) [inline guides/deploy/README.md:376]: I would change this to be a bit more general? Like "edit your deployment YAML file"

File: guides/deploy/README.md [hunk 2]
```diff
@@ -371,6 +371,107 @@ akash \
...
+
+Akash Groups are translated into Kubernetes Deployments, this means that only a few fields from the Akash SDL are mutable. For example image, command, args, env and exposed ports can be modified, but compute resources and placement criteria cannot. 
+
+{% endhint %}
+
+  1. Update your deployment by running:
+  ```sh
+  akash tx deployment update deploy.yml --dseq $DSEQ --from $KEY_NAME --chain-id $CHAIN_ID --node $AKASH_NODE --fees=5000uakt
+  ```
+  After confirming your transaction, you should see a response similar to this:
```
Comment (hydrogen18) [inline guides/deploy/README.md:388]: Probably mention to look for `code: 0` in the output

File: guides/deploy/README.md (hunk 1 above)
Comment (rsum1) [reply guides/deploy/README.md]: agree. fixed

File: guides/deploy/README.md (hunk 2 above)
Comment (rsum1) [reply guides/deploy/README.md:388]: added

**Review by hydrogen18 — APPROVED** [review]

**Pull request merged**